awswrangler/athena/_write_iceberg.py

```diff
@@ -196,6 +205,7 @@ def to_iceberg(
     dtype: Optional[Dict[str, str]] = None,
     catalog_id: Optional[str] = None,
     schema_evolution: bool = False,
+    columns_comments: Optional[Dict[str, str]] = None,
```

Comment:
I think this parameter should be wrapped inside of `glue_table_settings`, like it is [here](https://github.com/aws/aws-sdk-pandas/blob/main/awswrangler/s3/_write_parquet.py#L337).